lightkurve/utils.py

```diff
@@ -511,3 +511,40 @@ def detect_filetype(header):
     # if one of them is Undefined we expect `.lower()` to yield an AttributeError.
     except (KeyError, AttributeError):
         return None
+
+
+def boolean_mask_to_bitmask(aperture_mask):
+    """Takes in an aperture_mask and returns a Kepler-style bitmask
+
+    Parameters
+    ----------
+    aperture_mask : array-like
+        2D aperture mask. The mask can be either a boolean mask or an integer
+        mask mimicking the Kepler/TESS convention; boolean or boolean-like masks
+        are converted to the Kepler/TESS conventions.
+
+    Returns
+    -------
+    bitmask : numpy uint8 array
+        A bitmask mimicking the Kepler/TESS convention
```

Comment:
We will want the docstring to describe exactly what is being set in the returned mask, i.e. `3` where `aperture_mask == 1` and 1 otherwise?  We don't implement the full convention etc.

Reply:
fixed in 980c8d0